fontforgeexe/startui.c

```diff
@@ -1248,12 +1248,12 @@ int fontforge_main( int argc, char **argv ) {
     /*  the window around, which I can determine if I have a positioned */
```

Comment:
This is kind of contradicting this comment now...

Reply:
1250: `    /* Actually I don't care any more */`

That's George, lol. Maybe delete the whole comment block?